internal/pkg/action/operator_uninstall.go

```diff
@@ -107,6 +140,11 @@ func (u *OperatorUninstall) Run(ctx context.Context) error {
 		}
 	}
 
+	// If no objects were cleaned up, it means the package was not found
+	if subObj == nil && csvObj == nil && len(crds) == 0 {
+		return &ErrPackageNotFound{u.Package}
```

Comment:
Since we know `subObj` is never nil by this point (right?), I think we can completely remove this conditional block and use `ErrPackageNotFound` in line 65.